vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -106,11 +111,32 @@ void LedgerImpl::OnWalletInitializedInternal(
 void LedgerImpl::Initialize(ledger::InitializeCallback callback) {
   DCHECK(!initializing_);
   initializing_ = true;
+
+  initialize_callback_ = callback;
+
+  confirmations::_environment = ledger::_environment;
```

Comment:
can you move this all this code in `InitializeConfirmations` so that we have clear separation

Reply:
Fixed